cmd/blockchaincmd/create.go

```diff
@@ -558,3 +542,31 @@ func checkInvalidSubnetNames(name string) error {
 	}
 	return nil
 }
+
+func setSidecarValidatorManageOwner(sc *models.Sidecar, createFlags CreateFlags) error {
+	var err error
+	// if validatorManagerOwner flag is used, we get the C Chain address of the key used
+	if createFlags.validatorManagerOwner != "" {
```

Comment:
if I understand this code correctly clearer logic should be
* if `createFlags.validatorManagerOwner` empty - get it using getValidatorContractManagerAddr
* else `validateValidatorManagerOwnerFlag(createFlags.validatorManagerOwner) - verify user input  
followed by `sc.ValidatorManagerOwner = createFlags.validatorManagerOwner`

Reply:
removed comment and cleaned up the logic